src/main/webapp/app/exam/manage/student-exams/student-exams.component.html

```diff
@@ -23,12 +23,6 @@ <h2>
                     }
                     <span jhiTranslate="artemisApp.studentExams.generateMissingStudentExams"></span>
                 </button>
-                <button id="startExercisesButton" class="btn btn-primary mt-1" (click)="startExercises()" [disabled]="isLoading || isExamStarted || exercisePreparationRunning">
```

Comment:
This button is also mentioned [here](https://docs.artemis.cit.tum.de/user/exams/instructors_guide.html#manage-student-exams) in the documentation. The text there should be adjusted to the new behavior of Artemis and the screenshot of the button `Prepare exercise start` should be also removed.